src/main/java/cz/klemsa/tkbrew/Config.java

```diff
@@ -0,0 +1,22 @@
+package cz.klemsa.tkbrew;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class Config {
+
+
+    // --------------------------------------------------
+    // - LOG
+    // --------------------------------------------------
+    // TODO: Špatně... Když to někde zawiruju, bude dávat log třídu Config a ne od kud je log zavolán...
```

Comment:
Tady si můžeš zjednodušit práci, použitím lomboku.
čekni tohle:
https://howtodoinjava.com/spring-boot2/logging/logging-with-lombok/
Přidáš přes mvn dependency pro lombok, a pak už stačí jen nad jakoukoliv classou přidat
@Slf4j, a můžeš v ní používat    log.metody...() a nestaráš se o vytváření logů pro specifické třídy - jejich názvy budou použíté automaticky, při výpisech.